spec/features/brexit_checker/travel_question_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "spec_helper"
+
+RSpec.feature "Brexit Checker travel question", type: :feature do
+  scenario "User's living question choice is not displayed as a travel question option" do
```

Comment:
Just a minor thing, but the phrasing sounds like the 'feature' is a 'bug'! How about 'filter options by location'?

Reply:
👍